specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2344 @@
+      "in": "query",
+      "description": "Currently, only Afrikaans (‘af’), Albanian (‘sq’), Asturian (‘ast’), Basque (‘eu’), Bislama (‘bi’), Breton (‘br’), Catalan (‘ca’), Cebuano (‘ceb’), Chamorro (‘ch’), Cornish (‘kw’), Corsican (‘co’), Crimean Tatar - Latin script(‘crh’), Czech (‘cs’), Danish (‘da’), Dutch (‘nl’), English ('en'), Estonian (‘et’), Fijian (‘fj’), Filipino (‘fil’), Finnish (‘fi’), French (‘fr’), Friulian (‘fur’), Galician (‘gl’), German (‘de’), Gilbertese (‘gil’), Greenlandic (‘kl’), Haitian Creole (‘ht’), Hani (‘hni’), Hmong Daw (‘mww’), Hungarian (‘hu’), Indonesian (‘id’), Interlingua (‘ia’), Inuktitut (‘iu’), Irish (‘ga’), Italian (‘it’), Japanese (‘ja’), Javanese (‘jv’), Kabuverdianu (‘kea’), Kachin (‘kac’), Kara-Kalpak (‘kaa’), Kashubian (‘csb’), Khasi (‘kha’), Korean (‘ko’), Kurdish - Latin script (‘ku’), K’iche’ (‘quc’), Luxembourgish (‘lb’), Malay (‘ms’), Manx (‘gv’), Neapolitan (‘nap’), Norwegian (‘no’), Occitan (‘oc’), Polish (‘pl’), Portuguese (‘pt’), Romansh (‘rm’), Scots (‘sco’), Scottish Gaelic (‘gd’), simplified Chinese (‘zh-Hans’), Slovenian (‘sl’), Spanish (‘es’), Swahili (‘sw’), Swedish (‘sv’), Tatar - Latin script (‘tt’), Tetum (‘tet’), traditional Chinese (‘zh-Hant’), Turkish (‘tr’), Upper Sorbian (‘hsb’), Uzbek (‘uz’), Volapük (‘vo’), Walser (‘wae’), Western Frisian (‘fy’), Yucatec Maya (‘yua’), Zhuang (‘za’) and Zulu (‘zu’) are supported (print – seventy-three languages and handwritten – English only). Layout supports auto language identification and multi language documents, so only provide a language code if you would like to force the documented to be processed as that specific language.",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "af",
+        "ast",
+        "bi",
+        "br",
+        "ca",
+        "ceb",
+        "ch",
+        "co",
+        "crh",
+        "cs",
+        "csb",
+        "da",
+        "de",
+        "en",
+        "es",
+        "et",
+        "eu",
+        "fi",
+        "fil",
+        "fj",
+        "fr",
+        "fur",
+        "fy",
+        "ga",
+        "gd",
+        "gil",
+        "gl",
+        "gv",
+        "hni",
+        "hsb",
+        "ht",
+        "hu",
+        "ia",
+        "id",
+        "it",
+        "iu",
+        "ja",
+        "jv",
+        "kaa",
+        "kac",
+        "kea",
+        "kha",
+        "kl",
+        "ko",
+        "ku",
+        "kw",
+        "lb",
+        "ms",
+        "mww",
+        "nap",
+        "nl",
+        "no",
+        "oc",
+        "pl",
+        "pt",
+        "quc",
+        "rm",
+        "sco",
+        "sl",
+        "sq",
+        "sv",
+        "sw",
+        "tet",
+        "tr",
+        "tt",
+        "uz",
+        "vo",
+        "wae",
+        "yua",
+        "za",
+        "zh-Hans",
+        "zh-Hant",
+        "zu"
+      ],
+      "x-ms-enum": {
+        "name": "Language",
+        "modelAsString": true
+    },
```

Comment:
missing closing `}`

Reply:
👍  thanks so much for catching this. These are painful to find.